assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -35,6 +35,7 @@ import BackgroundColorSettings from './background-color-settings';
 import PageSettings from './page-settings';
 import BackgroundMediaSettings from './background-media-settings';
 import AnimationSettings from './animation-settings';
+import RemovePageSetting from './remove-button-setting';
```

Comment:
File name should follow component name.

Reply:
Also it‘s not really a setting, just a button.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3535/commits/555d351fcbdb18a9695bad51080802bf94361785